pytorch_lightning/trainer/supporters.py

```diff
@@ -76,3 +76,17 @@ def _agg_memory(self, how: str):
                 return getattr(self.memory, how)()
             else:
                 return getattr(self.memory[:self.current_idx], how)()
+
+
+class Accumulator(object):
```

Comment:
```suggestion
class Accumulator(torch.nn.Module):
```

making this a torch.nn.Module allows to use `.to()` and other device/dtype modifications as well